pkg/apiserver/resthandler.go

```diff
@@ -736,6 +737,10 @@ func setListSelfLink(obj runtime.Object, req *restful.Request, namer ScopeNamer)
 }
 
 func getPatchedJS(contentType string, originalJS, patchJS []byte, obj runtime.Object) ([]byte, error) {
+	// Remove "; charset=" if included in header.
```

Comment:
Effectively this is broken if people send anything other than UTF-8. Should we check? Maybe just add a TODO?